### PR TITLE
Init Mealz doc

### PR DESCRIPTION
**Pull request by ThibaultMiam**

### CODE REVIEW

**Comment by ThibaultMiam** [comment]
* **#5** <a href="https://app.graphite.dev/github/pr/miamtech/mealz-documentation/5?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `develop`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @ThibaultMiam and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: docs/android/overview/supplierInit.md [hunk 1]
```diff
@@ -0,0 +1,176 @@
...
+
+### initialValue
+
+When Mealz is launching, we need to know the state of the current customer basket.
+Whenever this basket changes, we will refresh the Mealz SDK.
+
+<BasketService.InitializeBasket />
+
+:::danger
+IMPORTANT: passing an empty list to Mealz SDK on initialization when there are actual products in the basket will lead to unexpected side effects. If this is done, products in the user's basekt will appear as removed by the user in the Mealz basket.
```
Comment (diarmuidmcg) [inline docs/android/overview/supplierInit.md:97]:
```suggestion
IMPORTANT: passing an empty list to Mealz SDK on initialization when there are actual products in the basket will lead to unexpected side effects. If this is done, products in the user's basket will appear as removed by the user in the Mealz basket.
```

File: docs/android/overview/supplierInit.md [hunk 2]
```diff
@@ -0,0 +1,176 @@
...
+:::
+
+### receive
+
+The last step of the `basketService` is to implement `receive`.
+This will update make sure both baskets are connected & up to date.
```
Comment (diarmuidmcg) [inline docs/android/overview/supplierInit.md:103]:
```suggestion
This will make sure both baskets are connected & up to date.
```

**Review by diarmuidmcg — CHANGES_REQUESTED** [review]

**Comment by ThibaultMiam** [comment]
### Merge activity

* **May 28, 9:57 AM EDT**: @ThibaultMiam started a stack merge that includes this pull request via [Graphite](https://app.graphite.dev/github/pr/miamtech/mealz-documentation/5).

**Pull request merged**